protocols/luci-proto-wireguard/htdocs/luci-static/resources/protocol/wireguard.js

```diff
@@ -114,9 +139,24 @@ return network.registerProtocol('wireguard', {
 		o.validate = validateBase64;
 		o.rmempty = false;
 
-		o = ss.option(form.Value, 'preshared_key', _('Preshared Key'), _('Optional. Base64-encoded preshared key. Adds in an additional layer of symmetric-key cryptography for post-quantum resistance.'));
+		var preshared_key_files = {},
+		    comment = ' Adds in an additional layer of symmetric-key cryptography for post-quantum resistance.';
```

Comment:
Maybe we could wrap this in _('.....') and add this to lines with `, _('....') + comment )`?
I have not verified if this works but if we would have only one additional translation and not extend two with this additional text.